pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -145,6 +145,10 @@ func (o *createChangefeedOptions) addFlags(cmd *cobra.Command) {
 
 // complete adapts from the command line args to the data and client required.
 func (o *createChangefeedOptions) complete(ctx context.Context, f factory.Factory, cmd *cobra.Command) error {
+	if o.commonChangefeedOptions.sinkURI == "" {
```

Comment:
why only move this one, seems have the same logic
```
func (o *createChangefeedOptions) validateTargetTs() error {
	if o.commonChangefeedOptions.targetTs > 0 && o.commonChangefeedOptions.targetTs <= o.startTs {
		return errors.Errorf("target-ts %d must be larger than start-ts: %d", o.commonChangefeedOptions.targetTs, o.startTs)
	}
	return nil
}
```

Reply:
Please add some unit tests for it.

Reply:
ping @3AceShowHand no update for 7 days